src/data-display/dynamic/dynamic-field/templates/list/dict-list/PDictList.vue

```diff
@@ -1,32 +1,41 @@
 <template>
     <ul class="p-dict-list">
-        <li v-for="(value, key) in dict" :key="key">
-            {{ key }} : {{ value }}
+        <li v-for="(value, key, index) in dict" :key="`dict-list-${key}-${index}`">
+            <span class="key">{{ key }}: </span>
```

Comment:
In Figma, designers used `tags` element. 
To avoid design fragmentation, I think it must be developed using the `p-tag` component.

---
피그마를 보면 디자이너분들이 태그 요소를 사용했습니다.
디자인 파편화를 피하려면 p-tag 컴포넌트를 사용하여 개발되어야 할 것 같습니다.